client/src/App.js

```diff
@@ -3,42 +3,61 @@ import DateSelector from './components/DateSelector';
 import Guests from './components/Guests';
 import { library } from '@fortawesome/fontawesome-svg-core';
 import axios from 'axios';
+import faker from 'faker';
 import { StickyContainer, Sticky } from 'react-sticky';
-import { faArrowRight, faIgloo, faPlusCircle, faMinusCircle} from '@fortawesome/free-solid-svg-icons';
+import { faStarHalf, faStar, faArrowRight, faIgloo, faPlusCircle, faMinusCircle} from '@fortawesome/free-solid-svg-icons';
+const moment = require('moment')
```

Comment:
since you're using imports, i would be consistent and use `import moment from 'moment';`

Reply:
@kogisu good call, dont rememeber why but I had some issues importing it previously. I re-adjust to use an import stmt and it seems to be working now. Thanks for catching